src/practitest_firecracker/practitest.clj

```diff
@@ -10,7 +10,7 @@
 
 ;; ===========================================================================
 ;; api version
-(def ^:const fc-version "1.4.0")
+(def ^:const fc-version "1.1.3")
```

Comment:
Why? version number should always increase, why did you decrease it?

Reply:
I think the old version was a mistake